src/types.d.ts

```diff
@@ -1,56 +1,75 @@
 
-declare module '@google-cloud/common' {
```

Comment:
Why is this line being deleted?

Reply:
I'm using the `types.d.ts` file directly to solve a problem where Typescript code that uses this library would get an error during compilation stating that the types in `types.d.ts` could not be resolved.

Reply:
I see. So this seems to be the main motivation of this PR vs adding an installation test, right? The latter seems just a means to achieve the former. Can you mention the main purpose in the PR description?

BTW which types from `types.d.ts` cause compilations errors? Why are they needed from the client code?